src/js/main.js

```diff
@@ -486,4 +489,68 @@ Site.Map = {
 
 };
 
+
+Site.Minimap = {
+  minimapScale: .03,
+  indicatorPosition: false,
+  init: function() {
+    var _this = this;
+
+    // Set indicator element
+    _this.minimapIndicator = document.getElementById('minimap-indicator');
+
+    _this.bindClick();
+  },
+
+  bindClick: function() {
+    var _this = this;
+
+    // Bind click on minimap buttons
+    $('.minimap-button').on('click', function() {
+      // HANDLE IT
+      _this.handleClick(this);
+    });
+  },
+
+  handleClick: function(elem) {
+    var _this = this;
+
+    // Parse targets position JSON from data attr
+    var pos = JSON.parse($(elem).attr('data-pos'));
+    var col = pos.col;
+    var row = pos.row;
+
+    // Multiply positions by window sizes to get new map coordinates
+    var x = col * Site.Map.window.width * -1;
+    var y = row * Site.Map.window.height * -1;
+
+    // Move map to new coordinates
+    // moveIndicator is called by this function
+    Site.Map.moveMap(x,y);
+  },
+
+  moveIndicator: function(x,y) {
+    var _this = this;
+
+    // Multiply map position by minimap scale.
+    // Map moves to negative coordinates,
+    // Minimap indicator moves positive
+    var minimapX = x * -(_this.minimapScale);
+    var minimapY = y * -(_this.minimapScale);
+
+    // If we don't know the indicator's current postion we get it
+    if (!_this.indicatorPosition) {
+      _this.indicatorPosition = Site.Map.getMapPosition(_this.minimapIndicator);
```

Comment:
I think the use of `getMapPosition` to return the position of an element passed to it means the function should be renamed